services/bit/bit-components.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+const { metric } = require('../text-formatters')
+
+const ownerSchema = Joi.object({
+  payload: Joi.object({
+    totalComponents: Joi.number(),
+  }),
+}).required()
+
+module.exports = class bitComponents extends BaseJsonService {
+  async fetch({ owner, scope }) {
+    const url = `https://api.bit.dev/scope/${owner}/${scope}/`
+    return this._requestJson({
+      url,
+      schema: ownerSchema,
+      errorMessages: {
+        404: 'collection not found',
+      },
+    })
+  }
+
+  static render({ count }) {
+    return { message: metric(count), color: '#73398D' }
```

Comment:
I'm not familiar with the bit.dev service, but I take it there's a reason to always use the `#73398D` color? With other count/download/etc. related metrics, it is common for the badges to use a color scale (for example `downloadCount` from `color-formatters.js`). 

Do you think it would be valuable here to have a color scale based on the count value, or does the static `#73398D` color make more sense?

Reply:
you are right, I have changed it to dynamic color using downloadCount function.